array.c

```diff
@@ -5067,32 +5067,220 @@ rb_ary_clear(VALUE ary)
 
 /*
  *  call-seq:
- *     ary.fill(obj)                                 -> ary
- *     ary.fill(obj, start [, length])               -> ary
- *     ary.fill(obj, range)                          -> ary
- *     ary.fill {|index| block}                      -> ary
- *     ary.fill(start [, length]) {|index| block}    -> ary
- *     ary.fill(range) {|index| block}               -> ary
+ *    array.fill(obj) -> self
+ *    array.fill(obj, start) -> self
+ *    array.fill(obj, start , length) -> self
+ *    array.fill(obj, range) -> self
+ *    array.fill { |index| ... } -> self
+ *    array.fill(start) { |index| ... } -> self
+ *    array.fill(start , length) { |index| ... } -> self
```

Comment:
We can get rid of the space before the `,`:

```suggestion
 *    array.fill(start, length) { |index| ... } -> self
```